### PR TITLE
Feature/metadata languages

### PR DESCRIPTION
**Pull request by jesusgonzalezrivera**

### CODE REVIEW

**Comment by clementprdhomme** [comment]
@jesusgonzalezrivera When I try to edit a dataset, I get this error:

<p align="center">
NoMethodError: undefined method `symbolize_keys'
</p>

I tried this one: `/management/sites/base-site/datasets/445cfe30-4989-4ff3-8378-d31d6120abdc/dataset_steps/metadata/edit`.

Is my local config wrong?

BTW, did you notice any issue in the front-end? This review goes both ways.

**Comment by jesusgonzalezrivera** [comment]
Done!

**Comment by clementprdhomme** [comment]
@jesusgonzalezrivera As we discussed on Slack, can you make two last changes?

1. If the site doesn't have any languages set, you would return all of them
2. The «Metadata» step being the last one, you would remove the «Continue» button of the bottom bar

**Comment by clementprdhomme** [comment]
Oh and @santostiago, we're still awaiting for you review.

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-8.9.4
+8.11.3
```
Comment (santostiago) [inline .nvmrc]: So we need to change the version of NVM?

Comment (clementprdhomme) [reply .nvmrc]: No, this changes the version of node. I needed it in order to update a dependency.

**Comment by jesusgonzalezrivera** [comment]
Fix all the comments ^^

File: app/models/dataset.rb [hunk 2]
```diff
@@ -123,7 +123,7 @@ def attributes
   end
 
   def connector_url=(value)
-    if !@connector.eql? 'arcgis' or value.include? 'f=pjson'
+    if !@connector.eql?('arcgis') || value.include?('f=pjson')
```
Comment (santostiago) [inline app/models/dataset.rb]: 👍

**Pull request merged**